main.py

```diff
@@ -9,4 +9,10 @@ def read_root():
 
 @app.get("/explain/{term}")
 def explain(term: str):
-    return controller.explain(term)
\ No newline at end of file
+    return controller.explain(term)
+
+@app.get("/define/{term}")
+def define(term: str):
+    term_file = "data/term_to_cui.json"
```

Comment:
Since these are defined as static paths in retrieval.py do we need these here? Might be able to simplify this